messaging/src/main/java/org/axonframework/eventhandling/SubscribingEventProcessor.java

```diff
@@ -198,6 +205,12 @@ public Builder processingStrategy(EventProcessingStrategy processingStrategy) {
             return this;
         }
 
```

Comment:
I would expect JavaDoc on this method. Would you mind adding it in a similar format as the other builder methods?
Note that it's important to specify what the default is, if it's not provided.

Similarly, the `Builder`'s class level JavaDoc should also be updated to point out the `TransactionManager`, as well as the JavaDoc of the `builder()` method.